packages/components/src/components/form/component.tsx

```diff
@@ -119,4 +126,13 @@ export class KolForm implements API {
 		this.validateRequiredText(this._requiredText);
 		this.validateErrorList(this._errorList);
 	}
+
+	public componentDidRender() {
+		if (this._errorList && this._errorList.length > 0) {
+			const errorElements = this.hostElement && this.hostElement.shadowRoot?.querySelector<HTMLElement>('#first-error-element');
```

Comment:
```suggestion
			const errorElement = this.hostElement && this.hostElement.shadowRoot?.querySelector<HTMLElement>('#first-error-element');
```
Ein Element, demnach Singular